package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "weather_app",
+  "version": "1.0.0",
+  "description": "The aim of this project was to create a weather app where users can input a city and view the weather type and temperature of that city. The original project requirements can be found on [The Odin Project](https://www.theodinproject.com/courses/javascript/lessons/weather-app).",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

Comment:
setup script to work properly with webpack ![see](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/scripts_requirement.png)

`"scripts": {`
    `"dev": "webpack --mode development",`
    `"build": "webpack --mode production",`
    `"start": "webpack-dev-server --mode development --open" // optional`
  `},`